Tests/TestingApps/SimpleApp/SimpleApp/Views/SampleChart.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  SampleChart.swift
+//  SimpleApp
+//
+//  Created by Nacho Soto on 7/17/23.
+//
+
+import SwiftUI
+
+struct BarChartView: View {
```

Comment:
This is just for the `SimpleApp`, to have a "main app view" that's not too empty and makes having mini-paywalls there make a bit more sense.